### PR TITLE
[14.0] edi: backend archive (port from v13) + integrate w/ edi_endpoint

### PR DESCRIPTION
**Pull request by simahawk**
https://github.com/OCA/edi/pull/623

### CODE REVIEW

File: edi_endpoint_oca/models/edi_backend.py [hunk 1]
```diff
@@ -16,10 +16,16 @@ class EDIBackend(models.Model):
     )
     endpoints_count = fields.Integer(compute="_compute_endpoints_count")
 
-    @api.depends("endpoint_ids")
+    @api.depends("endpoint_ids.active")
     def _compute_endpoints_count(self):
+        data = self.env["edi.endpoint"].read_group(
+            [("backend_id", "in", self.ids), ("active", "=", True)],
```
Comment (SilvioC2C) [inline edi_endpoint_oca/models/edi_backend.py:22]: `("active", "=", True)` shouldn't be necessary

**Review by SilvioC2C — APPROVED** [review]
LGTM, one minor comment

File: edi_endpoint_oca/models/edi_backend.py (hunk 1 above)
Comment (simahawk) [reply edi_endpoint_oca/models/edi_backend.py:22]: It's just more explicit.

**Review by TDu — APPROVED** [review]

**Pull request merged**